File: Kotlin/1009.kt
import java.util.*

fun main(args: Array<String>) {
    val name = readLine()!!
    val salary = readLine()!!.toFloat()
    val sales = readLine()!!.toFloat()

    println("TOTAL = R$ " + "%.2f".format(salary + 0.15*sales))
}

File: C/1009.c
#include <stdio.h>
 
int main() {
    char name[15];
    double salary, bonus, finalsalary;
    gets(name);
    scanf("%lf %lf", &salary, &bonus);
    finalsalary = salary+(bonus*0.15);
    printf("TOTAL = R$ %.2lf\n", finalsalary); 
    return 0;
}